### ISSUE TITLE
Not correct segmentation image

### ISSUE THREAD

**Issue by OliverVarnce** [sangyun884/HR-VITON#41]
I have prepared segmentation image in CIHP_PGN but when I execute it in HR-VITON I have this error:

ValueError: operands could not be broadcast together with shapes (1024,768) (1024,768,3)

**Comment by lujiazho** [comment]
I guess it's because you didn't convert it to P mode

```
Image.open("./output/cihp_parsing_maps/00008_00_vis.png").convert('P')
```

**Comment by OliverVarnce** [comment]
Yes! You are right. I have already figured out this issue. Thanks for tips.
